GHzDACs/ghz_fpga_server.py

```diff
@@ -1900,6 +2046,24 @@ def dac_bringup(self, c, lvdsOptimize=False, lvdsSD=None, signed=True,
                 ansDAC.append((key,val))
             ans.append(tuple(ansDAC))
         returnValue(ans)
+
+    @setting(1313, 'DAC Serial', cmd='w', pkts='*w', returns='?')
+    def dac_serial(self, c, cmd, pkts):
+        dev = self.selectedDAC(c)
+        ans = yield dev.runSerial(cmd, pkts)
+        returnValue(ans)
+
+    @setting(1314, 'Print Context')
+    def print_context(self, c):
```

Comment:
this and the next seem like debug settings; I'd suggest calling them something like `debug_print_context`/`debug_clear_ethernet` and putting a gap in the setting numbering, e.g. make these something like settings 1000000 and 1000001.